webapp/src/pages/ChatsPage/template/ChatsPageTemplate.tsx

```diff
@@ -32,20 +34,27 @@ export default function ChatsPageTemplate({
   buttonIconVariant,
   buttonLabel,
 }: ChatPageTemplateProps) {
+  const { userStatus } = useUserStatus();
+  const { userFriends } = useFriend();
   const navigate = useNavigate();
   const overridePermissions =
     buttonUrl.slice(0, ADMIN_URL.length) === ADMIN_URL;
-  const mapChatToRow = (chatroom: Chatroom): RowItem => {
+  const mapChatToRow = (chatroom: GenericChat): RowItem => {
     return {
       iconVariant: IconVariant.ARROW_FORWARD,
       avatarProps: {
         url: `${AVATAR_EP_URL}/${chatroom.avatarId}`,
+        XCoordinate: chatroom.avatarX,
+        YCoordinate: chatroom.avatarY,
+        status:
+          chatroom.url.slice(0, CHATROOM_URL.length) !== CHATROOM_URL &&
+          userFriends(chatroom.id)
+            ? userStatus(chatroom.id)
+            : undefined,
```

Comment:
As commented above, rather than hardcoding the url in the query, we could attach some kind of metadata to the retrieved entity to identify if the message comes from a chatroom or a one-to-one. That way we could safely construct the destination url and the status

Reply:
done